presentation/src/main/java/com/yapp/presentation/ui/member/main/MemberMain.kt

```diff
@@ -72,6 +91,9 @@ fun MemberMain(
                                 tab = uiEffect.tab
                             )
                         }
+                        MemberMainContract.MemberMainUiSideEffect.NavigateToTeam -> {
+                            navigateToTeam.invoke()
```

Comment:
```suggestion
                            navigateToTeam()
```

`invoke()` 안붙여도 될 것 같아요!